### PR TITLE
fix-styles

### PR DESCRIPTION
**Pull request by Jula000**
first fix

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,9 +10,11 @@
   },
   "dependencies": {
     "@next/font": "^14.2.4",
-    "next": "14.2.4",
-    "react": "^18",
-    "react-dom": "^18",
+    "@splidejs/react-splide": "^0.7.12",
+    "@splidejs/splide": "^4.1.4",
```
Comment (yevhenii-horbenko) [inline package.json]: I'm not sure you're using these packages
Are you sure we need them?

File: src/app/globals.css [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+@keyframes marquee {
+  0% {
+    transform: translateX(100%);
+  }
+  100% {
+    transform: translateX(-50%);
+  }
+}
+
+@media screen and (min-width: 680px) {
```
Comment (yevhenii-horbenko) [inline src/app/globals.css]: I'm not sure that we need these classes accessible in global scope
let's discuss

File: src/components/AboutPage/AboutSection/index.tsx [hunk 3]
```diff
@@ -2,68 +2,54 @@ import React from "react";
...
+      <div className="flex flex-wrap mb-8 justify-center">
+        <div className="grid grid-cols-2 md:grid-cols-3 lg:grid-cols-6 gap-4 mb-8">
+          {[
+            { count: "15+", label: "Years in Business" },
+            { count: "500+", label: "Happy Clients" },
+            { count: "10+", label: "Photography Awards" },
+            { count: "05+", label: "International Shoots" },
+            { count: "10,000+", label: "Social Media Followers" },
+            { count: "90%", label: "Client Retention Rate" },
+          ].map((item, index) => (
```
Comment (yevhenii-horbenko) [inline src/components/AboutPage/AboutSection/index.tsx]: it's better to put this array outside of render

Comment (yevhenii-horbenko) [reply src/components/AboutPage/AboutSection/index.tsx]: but love the idea
much less repetitive code!

File: src/components/ContactPage/ContactSection/index.tsx [hunk 4]
```diff
@@ -16,8 +16,13 @@ const ContactSection: React.FC = () => {
...
         </div>
-        <div className="md:w-auto md:ml-4 flex justify-center md:justify-end">
-          <SocialMediaButtons />
+        <div
+          className="w-1/2 h-80 rounded-3xl overflow-hidden bg-cover bg-center relative"
+          style={{ backgroundImage: "url('/kytsya.jpg')" }}
+        >
+          <div className="bottom-1/2 -right-4 bg-main-color px-4 py-4 rounded-full absolute">
+            <SocialMediaButtons className="rounded-full border border-gray-600 px-2 py-2" />
+          </div>
```
Comment (yevhenii-horbenko) [inline src/components/ContactPage/ContactSection/index.tsx]: I'm  not sure you need absolute here, let's discuss

File: src/components/HomePage/ServiceSection/index.tsx [hunk 5]
```diff
@@ -1,31 +1,40 @@
...
   "Skilled photographers who know how to seize the moment.",
   "A mix of candid and posed shots for a comprehensive story.",
   "Quick turnaround for you to relive the day's highlights.",
 ];
 
-const title: string = "Events";
-
-const description: string =
+const title = "Events";
+const description =
```
Comment (yevhenii-horbenko) [inline src/components/HomePage/ServiceSection/index.tsx]: why do you store these two variables outside the component
I'd just use literal values

File: src/components/PortfolioPage/PortfolioSection/index.tsx [hunk 6]
```diff
@@ -13,28 +13,45 @@ const PortfolioSection: React.FC = () => {
...
+            className="relative w-full h-96 rounded-3xl overflow-hidden bg-cover bg-center"
+            style={{ backgroundImage: "url('/kytsya.jpg')" }}
+          ></div>
+        </section>
         <h2 className="mb-4 uppercase text-gray-400 text-center">
           brands i have worked with
         </h2>
       </section>
-      <div className="flex flex-wrap justify-center bg-zinc-900">
+      <div className="flex flex-wrap justify-center  bg-zinc-900">
```
Comment (yevhenii-horbenko) [inline src/components/PortfolioPage/PortfolioSection/index.tsx]: you have an unnecessary space here

File: src/components/ServicesPage/ServicesSection/index.tsx [hunk 7]
```diff
@@ -3,16 +3,31 @@ import React from "react";
 const ServicesSection = () => {
   return (
     <section className="container mx-auto px-4 py-16 text-gray-400">
-      <p className="uppercase text-base text-gray-400">Services</p>
-      <h2 className="text-3xl font-bold mb-4 text-white">
-        Diverse Photography Offerings
-      </h2>
+      {" "}
```
Comment (yevhenii-horbenko) [inline src/components/ServicesPage/ServicesSection/index.tsx]: why do you have an empty space here

File: src/components/PortfolioPage/PortfolioSection/index.tsx [hunk 8]
```diff
@@ -13,28 +13,45 @@ const PortfolioSection: React.FC = () => {
...
+              Portfolio
+            </p>
+            <h1 className="xl:text-3xl 2xl:text-4xl font-bold mb-4">
+              VISUAL POETRY IN PIXELS
+            </h1>
+          </div>
+          <div
+            className="h-20 lg:w-7/12 xl:w-4/6 rounded-3xl bg-cover bg-center"
+            style={{ backgroundImage: "url('/kytsya.jpg')" }}
+          ></div>
```
Comment (yevhenii-horbenko) [inline src/components/PortfolioPage/PortfolioSection/index.tsx]: let's move it to a separate component

File: src/components/PortfolioPage/PortfolioSection/index.tsx [hunk 9]
```diff
@@ -13,28 +13,45 @@ const PortfolioSection: React.FC = () => {
...
           Each photograph in this portfolio is a narrative, a frozen moment in
           time, and a testament to the artistry and passion poured into every
           frame. Explore the diverse tapestry of stories I've had the privilege
           to capture and witness the world through my lens.
         </p>
+        <section className="container mx-auto px-4 mb-16">
+          <div
+            className="relative w-full h-96 rounded-3xl overflow-hidden bg-cover bg-center"
+            style={{ backgroundImage: "url('/kytsya.jpg')" }}
+          ></div>
```
Comment (yevhenii-horbenko) [inline src/components/PortfolioPage/PortfolioSection/index.tsx]: let's move it to a separate component

File: src/components/ServicesPage/ServicesSection/index.tsx [hunk 10]
```diff
@@ -3,16 +3,31 @@ import React from "react";
...
+        <div className="flex-cols">
+          <p className="uppercase text-base text-gray-400">Services</p>
+          <h2 className="text-3xl font-bold mb-4 text-white">
+            Diverse Photography Offerings
+          </h2>
+        </div>
+        <div
+          className="h-20 w-8/12 rounded-3xl overflow-hidden bg-cover bg-center self-start"
+          style={{ backgroundImage: "url('/kytsya.jpg')" }}
+        />
```
Comment (yevhenii-horbenko) [inline src/components/ServicesPage/ServicesSection/index.tsx]: let's move it to a separate component

File: src/components/ServicesPage/ServicesSection/index.tsx [hunk 11]
```diff
@@ -3,16 +3,31 @@ import React from "react";
...
         Unlock the full spectrum of professional photography services tailored
         to your vision. From timeless portraits to captivating event coverage, I
         bring a unique blend of creativity and technical expertise to each
         project
       </p>
+      <section className="container mx-auto px-4 mb-4 pt-8">
+        <div
+          className="relative w-full h-96 rounded-3xl overflow-hidden bg-cover bg-center"
+          style={{ backgroundImage: "url('/kytsya.jpg')" }}
+        ></div>
```
Comment (yevhenii-horbenko) [inline src/components/ServicesPage/ServicesSection/index.tsx]: let's move it to a separate component

File: src/components/ServicesPage/ServicesSection/index.tsx [hunk 12]
```diff
@@ -3,16 +3,31 @@ import React from "react";
...
+      </section>
     </section>
```
Comment (yevhenii-horbenko) [inline src/components/ServicesPage/ServicesSection/index.tsx]: you have a section inside section, that's probably not okay with semantic html

File: src/components/common/Footer.tsx [hunk 13]
```diff
@@ -1,100 +1,96 @@
...
+                </ul>
+              </div>
+              <div className="w-1/2 sm:w-1/4 lg:w-auto mb-6 lg:mb-0">
+                <h3 className="font-semibold text-gray-600">SERVICES</h3>
+                <ul className="mt-2 space-y-2">
+                  <li className="border-b border-gray-600 pb-1">Portraits</li>
+                  <li className="border-b border-gray-600 pb-1">Events</li>
+                  <li className="border-b border-gray-600 pb-1">Commercial</li>
+                </ul>
+              </div>
```
Comment (yevhenii-horbenko) [inline src/components/common/Footer.tsx]: you can probably use a component for these

File: src/components/common/Footer.tsx [hunk 14]
```diff
@@ -1,100 +1,96 @@
...
-                <li>Cloven's</li>
-                <li>Menvol</li>
-              </ul>
+          </div>
+        </div>
+        <div className="border-t border-gray-600 py-8">
+          <div className="container mx-auto px-4 flex flex-col md:flex-row justify-between items-center">
+            <div className="text-sm mb-4 md:mb-0">
+              <Link href="/terms">Terms & Conditions</Link> |{" "}
+              <Link href="/privacy">Privacy Policy</Link>
```
Comment (yevhenii-horbenko) [inline src/components/common/Footer.tsx]:
```suggestion
              <Link href="/terms">Terms & Conditions</Link>
              {" | "}
              <Link href="/privacy">Privacy Policy</Link>
```
I think this one looks a bit better

File: src/components/common/Marquee.tsx [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+    <>
+      <span className="mx-4 inline-block">EVENT PHOTOGRAPHY</span>
+      <span className="mx-4 inline-block">COMMERCIAL PHOTOGRAPHY</span>
+      <span className="mx-4 inline-block">PRODUCT PHOTOGRAPHY</span>
+      <span className="mx-4 inline-block">WEDDING PHOTOGRAPHY</span>
+      <span className="mx-4 inline-block">LANDSCAPE PHOTOGRAPHY</span>
+      <span className="mx-4 inline-block">BRANDING PHOTOGRAPHY</span>
+      <span className="mx-4 inline-block">PORTRAIT PHOTOGRAPHY</span>
+    </>
+  );
```
Comment (yevhenii-horbenko) [inline src/components/common/Marquee.tsx]: why do you have it as a separate variable?

File: src/components/common/ProjectList.tsx [hunk 16]
```diff
@@ -16,21 +17,35 @@ const ProjectList: React.FC<Props> = ({ projects }) => (
...
-            style={{ backgroundImage: `url(${project.image})` }}
-          />
-          <div className="mt-4 flex flex-col self-start">
-            <h2 className="text-xl font-semibold">{project.title}</h2>
-            <p className="text-sm">{project.date}</p>
-            <p className="mt-2 text-sm font-semibold text-blue-500">
-              VIEW PROJECT
-            </p>
+    <div className="flex overflow-hidden gap-6">
+      {projects?.map((project, index) => (
```
Comment (yevhenii-horbenko) [inline src/components/common/ProjectList.tsx]: projects should be required so optional chaining here is unnecessary

**Review by yevhenii-horbenko — CHANGES_REQUESTED** [review]

File: src/components/common/Marquee.tsx (hunk 15 above)
Comment (Jula000) [reply src/components/common/Marquee.tsx]: ? how to do it differently

File: src/components/ContactPage/ContactSection/index.tsx (hunk 4 above)
Comment (Jula000) [reply src/components/ContactPage/ContactSection/index.tsx]: with
image

Comment (Jula000) [reply src/components/ContactPage/ContactSection/index.tsx]: without 
image

File: src/app/globals.css [hunk 17]
```diff
@@ -1,3 +1,29 @@
...
+}
+
+.project-image {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
```
Comment (yevhenii-horbenko) [inline src/app/globals.css]: please use tailwind classes for it

File: src/components/ContactPage/ContactForm/index.tsx [hunk 18]
```diff
@@ -7,80 +7,90 @@ const ContactForm: React.FC = () => {
...
                 Feel free to reach out to us through various channels. We are
-                <br />
+                <br className="hidden md:block" />
                 available by phone, email, and social media for your
                 convenience.
               </p>
             </div>
-            <div className="flex md:flex-row md:items-center md:space-x-4">
-              <div className="flex items-center space-x-2 mr-10">
+            <div className="flex flex-col items-center  lg:flex-row md:items-center md:space-x-4">
```
Comment (yevhenii-horbenko) [inline src/components/ContactPage/ContactForm/index.tsx]: please remove double space

File: src/components/HomePage/HeroSection/index.tsx [hunk 19]
```diff
@@ -1,46 +1,42 @@
...
+          objectFit="contain"
         />
       </div>
+      <div className="flex flex-col uppercase font-semibold text-5xl z-10 self-start md:self-center">
+        <div className="flex items-center">
+          <span className="mr-2 font-bold">Let's</span>
+          <GlossyButton
+            onClick={handleClick}
+            className="py-2 md:py-3 px-6 md:px-12"
+          ></GlossyButton>
```
Comment (yevhenii-horbenko) [inline src/components/HomePage/HeroSection/index.tsx]: You don't need a closing tag here

File: src/components/common/Section.tsx [hunk 20]
```diff
@@ -9,7 +10,8 @@ const Section: React.FC<SectionProps> = ({ title, children }) => {
   return (
     <div className="border border-about-bg p-6 rounded-lg space-y-4">
       <div className="flex space-x-2">
-        <span className="text-blue-400">✦</span>
+        <span className="text-blue-400"></span>
```
Comment (yevhenii-horbenko) [inline src/components/common/Section.tsx]: you can probably remove this span

File: src/components/common/NavLink.tsx [hunk 21]
```diff
@@ -1,13 +1,28 @@
...
   return (
-    <Link href={href} className="text-gray-300 hover:text-white">
+    <Link
+      href={href}
+      className={`text-white ${
+        isActive
+          ? "font-bold bg-zinc-800 rounded-l-md rounded-r-md"
+          : "font-normal"
+      } : "font-normal"
+      } hover:text-zinc-400 ${className}`}
```
Comment (yevhenii-horbenko) [inline src/components/common/NavLink.tsx]: please refactor this class as it has invalid class names in it

File: src/components/common/Marquee.tsx [hunk 22]
```diff
@@ -0,0 +1,25 @@
...
+
+  return (
+    <div className="overflow-hidden whitespace-nowrap relative bg-zinc-500">
+      <div className="flex animate-marquee space-x-4">
+        <div className="flex space-x-4">{content}</div>
```
Comment (yevhenii-horbenko) [inline src/components/common/Marquee.tsx]: I'm not sure that you need this variable
You can use these items directly

File: src/components/common/Footer.tsx [hunk 23]
```diff
@@ -1,100 +1,109 @@
...
+                },
+                {
+                  title: "SERVICES",
+                  links: [
+                    { text: "Portraits", href: "#" },
+                    { text: "Events", href: "#" },
+                    { text: "Commercial", href: "#" },
+                  ],
+                },
+              ].map((section, index) => (
```
Comment (yevhenii-horbenko) [inline src/components/common/Footer.tsx]: Please move it to a separate variable

File: src/components/PortfolioPage/PortfolioSection/index.tsx [hunk 24]
```diff
@@ -13,19 +14,37 @@ const PortfolioSection: React.FC = () => {
...
+
+        <section className="container mx-auto px-4 mb-16">
+          <BackgroundImage src="/kytsya.jpg" />
+        </section>
+
+        <h2 className="mb-4 uppercase text-zinc-400 text-center">
           brands i have worked with
         </h2>
       </section>
+
```
Comment (yevhenii-horbenko) [inline src/components/PortfolioPage/PortfolioSection/index.tsx]:
```suggestion
```

File: src/components/PortfolioPage/PortfolioSection/index.tsx [hunk 25]
```diff
@@ -13,19 +14,37 @@ const PortfolioSection: React.FC = () => {
...
           time, and a testament to the artistry and passion poured into every
           frame. Explore the diverse tapestry of stories I've had the privilege
           to capture and witness the world through my lens.
         </p>
-        <h2 className="mb-4 uppercase text-gray-400 text-center">
+
+        <section className="container mx-auto px-4 mb-16">
+          <BackgroundImage src="/kytsya.jpg" />
+        </section>
+
```
Comment (yevhenii-horbenko) [inline src/components/PortfolioPage/PortfolioSection/index.tsx]:
```suggestion
```

File: src/components/PortfolioPage/PortfolioSection/index.tsx [hunk 26]
```diff
@@ -13,19 +14,37 @@ const PortfolioSection: React.FC = () => {
...
+        <p className="text-lg leading-relaxed mb-8 text-zinc-400">
           Step into a visual journey that encapsulates the essence of my lens.
           Each photograph in this portfolio is a narrative, a frozen moment in
           time, and a testament to the artistry and passion poured into every
           frame. Explore the diverse tapestry of stories I've had the privilege
           to capture and witness the world through my lens.
         </p>
-        <h2 className="mb-4 uppercase text-gray-400 text-center">
+
```
Comment (yevhenii-horbenko) [inline src/components/PortfolioPage/PortfolioSection/index.tsx]:
```suggestion
```

File: src/components/PortfolioPage/PortfolioSection/index.tsx [hunk 27]
```diff
@@ -13,19 +14,37 @@ const PortfolioSection: React.FC = () => {
...
+              VISUAL POETRY IN PIXELS
+            </h1>
+          </div>
+          <BackgroundImage
+            src="/kytsya.jpg"
+            className="h-20 lg:w-7/12 xl:w-4/6 self-start hidden md:block"
+            height="h-20"
+          />
+        </div>
+
```
Comment (yevhenii-horbenko) [inline src/components/PortfolioPage/PortfolioSection/index.tsx]:
```suggestion
```

**Review by yevhenii-horbenko — CHANGES_REQUESTED** [review]
looks great, I have a few comments though

File: src/components/PortfolioPage/PortfolioSection/index.tsx [hunk 28]
```diff
@@ -1,28 +1,52 @@
...
-  { name: "Infinity Parker", logo: "/brands/logo1.png", width: 251 },
-  { name: "Omega Million", logo: "/brands/logo2.png", width: 353 },
-  { name: "WheelApp", logo: "/brands/logo3.png", width: 204 },
-  { name: "High Country Club", logo: "/brands/logo4.png", width: 322 },
-  { name: "EdgeKart", logo: "/brands/logo5.png", width: 183 },
-];
+import BackgroundImage from "@/components/common/BackgroundImage";
 
 const PortfolioSection: React.FC = () => {
+  const [brands, setBrands] = useState<any[]>([]);
```
Comment (yevhenii-horbenko) [inline src/components/PortfolioPage/PortfolioSection/index.tsx]: please don't use any when possible

**Review by yevhenii-horbenko — CHANGES_REQUESTED** [review]

**Review by yevhenii-horbenko — APPROVED** [review]
you're doing amazing!

**Pull request merged**